hopsworks-common/src/main/java/io/hops/hopsworks/common/provenance/core/elastic/ProvElasticController.java

```diff
@@ -63,7 +63,8 @@
   private Settings settings;
   @EJB
   private ElasticClient client;
-  
+
+  @TransactionAttribute(TransactionAttributeType.NOT_SUPPORTED)
```

Comment:
@robzor92 we set NOT_SUPPORTED because it's used in a timer?

Reply:
Yes precisely